### PR TITLE
bump opendistroVersion to 1.6.1

### PR DESCRIPTION
**Pull request by kaituo**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,18 +6,15 @@ The Open Distro for Elasticsearch Anomaly Detection plugin enables you to levera
 
 Anomaly detection is using Random Cut Forest (RCF) algorithm for detecting anomalous data points.
 
-You should use anomaly detection plugin with Open Distro Alerting plugin [1.2-alpha](https://github.com/opendistro-for-elasticsearch/alerting). You can also create a monitor based on anomaly detector. A scheduled monitor run will trigger the anomaly detection plugin and collecting anomalies to trigger alerts based on custom trigger conditions.
+You should use anomaly detection plugin with the same version of Open Distro Alerting plugin (https://github.com/opendistro-for-elasticsearch/alerting). You can also create a monitor based on anomaly detector. A scheduled monitor run will trigger the anomaly detection plugin and collecting anomalies to trigger alerts based on custom trigger conditions.
```
Comment (ylwu-amzn) [inline README.md:9]: After decoupling, monitor will not trigger AD run. Can you update this?

Comment (yizheliu-amazon) [inline README.md:9]: should we mention dependency on job-scheduler somewhere?

Comment (ylwu-amzn) [reply README.md:9]: +1, we should mention that, like ISM https://github.com/opendistro-for-elasticsearch/index-management

Comment (yizheliu-amazon) [reply README.md:9]: As I recall, I have to install job-scheduler before installing AD to ES

Comment (kaituo) [reply README.md:9]: good catch.  updated.

Comment (kaituo) [reply README.md]: added.

**Pull request merged**